cpp/tests/lists/drop_list_duplicates_tests.cpp

```diff
@@ -14,174 +14,232 @@
  * limitations under the License.
  */
 
-#include <cudf/lists/drop_list_duplicates.hpp>
-
 #include <cudf_test/base_fixture.hpp>
 #include <cudf_test/column_wrapper.hpp>
+#include <cudf_test/type_lists.hpp>
+
+#include <cudf/lists/drop_list_duplicates.hpp>
+#include <cudf/utilities/type_dispatcher.hpp>
+
+#include <algorithm>
+#include <unordered_set>
 
-using float_type = float;
 using int_type   = int32_t;
-using INT_LCW    = cudf::test::lists_column_wrapper<int_type>;
-using FLT_LCW    = cudf::test::lists_column_wrapper<float_type>;
-using STR_LCW    = cudf::test::lists_column_wrapper<cudf::string_view>;
+using float_type = float;
+
+using LIST_COL_FLT = cudf::test::lists_column_wrapper<float_type>;
+using LIST_COL_STR = cudf::test::lists_column_wrapper<cudf::string_view>;
 
-template <bool equal_test, class LCW>
+auto constexpr m_NaN = -std::numeric_limits<float_type>::quiet_NaN();
+auto constexpr m_inf = -std::numeric_limits<float_type>::infinity();
+auto constexpr p_inf = std::numeric_limits<float_type>::infinity();
+auto constexpr p_NaN = std::numeric_limits<float_type>::quiet_NaN();
```

Comment:
So 'm_' stands for 'minus' and 'p_' stands for 'plus'.

Reply:
Sorry for this. I have updated them with more expressive names.